packages/flutter/lib/src/gestures/events.dart

```diff
@@ -790,6 +790,61 @@ class PointerUpEvent extends PointerEvent {
   );
 }
 
+/// A base type for events that correspond to a discrete pointer signal.
+///
+/// There are events that originate from the pointer but don't change the state
```

Comment:
Is this giving a justification for the existence of this class?  If so, you should say something the equivalent of "This class exists to..." somewhere in this paragraph. Otherwise it just sounds like a statement of fact that seems like a non-sequitur.